src/sql/workbench/contrib/commandLine/electron-browser/commandLine.ts

```diff
@@ -220,11 +219,11 @@ export class CommandLineWorkbenchContribution implements IWorkbenchContribution,
 		let profile = new ConnectionProfile(this._capabilitiesService, null);
 		// We want connection store to use any matching password it finds
 		profile.savePassword = true;
-		profile.providerName = args.provider ? args.provider : Constants.mssqlProviderName;
+		profile.providerName = args.provider ?? Constants.mssqlProviderName;
 		profile.serverName = args.server;
-		profile.databaseName = args.database ? args.database : '';
-		profile.userName = args.user ? args.user : '';
-		profile.authenticationType = args.integrated ? Constants.integrated : args.aad ? Constants.azureMFA : (profile.userName.length > 0) ? Constants.sqlLogin : Constants.integrated;
+		profile.databaseName = args.database ?? '';
+		profile.userName = args.user ?? '';
+		profile.authenticationType = args.authenticationType ?? Constants.integrated;
```

Comment:
I guess we'll just be fine with requiring outside callers to get the casing right too? All our internal code pretty much assumes that. Might be useful to do case insensitive check here and then convert to defined case-sensitive version in the profile.

Reply:
I'm not sure I'm a fan of that. We could explicitly say this is what we're expecting, these are the values that aren't going to change.

When I call any REST api for example, the REST api doesn't correct me if I make a casing mistake, it just flat out fails and tells me to go fix my stuff. That's kinda the philosophy I'm following here, if that makes sense?

Reply:
Yeah that's fine. I personally hate it when stuff requires me to have a specific casing - but I realize it's a pretty normal thing and the code is simpler because of it so it seems fine.

What happens if you give a auth type with invalid casing? The main problem I guess I see is that I highly doubt we handle it in any sane manner so it'd likely just fail in some obscure way along the line. But adding in special logic just for this case isn't super great either so I'm fine either way.

Reply:
While I was testing I actually made the mistake of incorrect casing. It simply pops up the connection pane with "unrecognized auth method" or something along those lines.